draft-ietf-ppm-dap.md

```diff
@@ -519,6 +519,26 @@ DAP has three major interactions which need to be defined:
 * Computing the results of a given measurement, specified in {{aggregate-flow}}
 * Collecting aggregated results, specified in {{collect-flow}}
 
+In this section, we discuss each of the resources and the messages used to act
+on these resources. A resource's path is resolved relative to a server's
+endpoint to construct a resource URI. A server's endpoint MAY contain
+arbitrarily many path components before the DAP resources.
+
+Resource paths are specified as templates like:
+
+~~~
+[role]/resource_type/{resource-id}
```

Comment:
Suggestion: For consistency with the other "variables" in paths, use curly braces instead of square braces. Unless, do you intend `{}` to denote a variable that is expanded into base64 encoding?
```suggestion
{role}/resource_type/{resource-id}
```

Reply:
No, I was just preserving the existing usage of `[leader]`, `[helper]` and `[aggregator]`. Using `{}` is more consistent, though.

Reply:
Consistency is a good idea, I think. The curly brackets look nicer.